src/util/string.js

```diff
@@ -0,0 +1,8 @@
+export const humanize = (string) => {
```

Comment:
This function is interesting. Do you mind writing some comments with at least one example to explain exactly what it does? Thanks.